jib-gradle-plugin/CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
+- Added an option `configurationName` to specify the name of the [Gradle Configuration](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.ConfigurationContainer.html) to use. The option can be lazily configured, for example, using Gradle `Provider` or `Property`. ([#3034](https://github.com/GoogleContainerTools/jib/pull/3034))
+```gradle
+    jib {
+      configurationName.set = 'myconfig'
```

Comment:
Ah, my bad. I think it's just
```suggestion
      configurationName = 'myconfig'
```
Right? (I think it worked in this way.)

Reply:
My bad, thank you. Just did a quick test and `configurationName = 'myConfig'` works.